API/Program.cs

```diff
@@ -158,7 +158,8 @@
     {
         context.HttpContext.Response.StatusCode = StatusCodes.Status429TooManyRequests;
         const string readMore =
-            "Read more about our rate limits at https://github.com/osu-tournament-rating/otr-wiki/blob/master/api/limits/en.md";
+            // TODO: Link to the API Terms of Service Document
+            "Read more about our rate limits at https://docs.otr.stagec.xyz/o-tr-api.html";
```

Comment:
```suggestion
            "Currently, information about API ratelimits is unavailable.";
```

I would rather not link a page that doesn't have this info.